JRecord/src/main/java/net/sf/JRecord/utilityClasses/Copy.java

```diff
@@ -442,4 +446,13 @@ public AbstractLine newLine() throws IOException {
 		}
 		
 	}
+
+	private static boolean isZeroString(String hexString) {
+		StringBuilder builder = new StringBuilder();
```

Comment:
no need to build a new string
for example:
```suggestion
       for (int i = 0; i < hexString.length(); i++) {
            if (hexString.charAt(i) != '0') {
                return false;
            }
        }
        return true;
```